src/components/look-controls.js

```diff
@@ -50,6 +51,24 @@ module.exports.Component = registerComponent('look-controls', {
     if (this.el.sceneEl.is('vr-mode')) { this.onEnterVR(); }
   },
 
+  setupMagicWindowControls: function () {
+    var magicWindowControls;
+    // Only on mobile devices and only enabled if DeviceOrientation permission has been granted.
+    if (utils.device.isMobile()) {
+      magicWindowControls = this.magicWindowControls = new THREE.DeviceOrientationControls(this.magicWindowObject);
+      if (typeof DeviceOrientationEvent === 'undefined' && DeviceOrientationEvent.requestPermission) {
+        magicWindowControls.enabled = false;
+        if (this.el.sceneEl.components['device-orientation-permission-ui'].premissionGranted) {
```

Comment:
there is a typo here premissionGranted => permissionGranted

Reply:
Good catch!